### PR TITLE
Update MIP113.md

### PR DESCRIPTION
**Pull request by 0xNix0**
MIP113 5.2.1 
Include “or to join an existing AVC” to the sentence. As stated in  5.2.1.2.4.

MIP113 5.2.3
Replace reference “4.5” for MIP113 5.2.1.2.4 and “4.4” for MIP101 2.5.3

MIP113 5.2.4.2
Replace reference “4.3” for MIP101 5.2.4.2

MIP113 5.2.1.1
Include context to 5.2.1.1 - "The condition must be met during all the mandate". As stated in ATL2.5.4.2

MIP113 5.2.1 
Replace "Ecosystem Actor " for "Alignment Conserver" - As stated in ATL2.5.3.1

MIP113 5.2.4.6.1A
Reclassify the RISK AVC as 'Inactive' for reasons stated in MIP113 5.2.4.2 - Place the RISK AVC in this category until the end of Q4, as justified in MIP 113 5.2.4.5

MIP113 5.2.4.2
Update reference from ”*4.3”* to ATL101 2.5.4.

### CODE REVIEW

File: MIP113/MIP113.md [hunk 1]
```diff
@@ -370,20 +370,20 @@ List of Unaffiliated AVC Members:
 
 #### 5.2.3
 
-An Unaffiliated AVC Member may join an AVC through the process described in 4.5 or start a new AVC through the process described in 4.4.
+An Unaffiliated AVC Member may join an AVC through the process described in MIP113 5.2.1.1 or start a new AVC through the process described in MIP101 2.5.3.
```
Comment (0xldr) [inline MIP113/MIP113.md]:
```suggestion
An Unaffiliated AVC Member may join an AVC through the process described in *5.2.1.1* or start a new AVC through the process described in *2.5.3* of The Atlas.
```

File: MIP113/MIP113.md [hunk 2]
```diff
@@ -312,11 +312,11 @@ AVC splits are disabled until the launch of NewChain.
 
 #### 5.2.1
 
-To become recognized as an Unaffiliated AVC Member, an Ecosystem Actor must post a recognition submission message publicly on the Maker Governance Forum.
+To become recognized as an Unaffiliated AVC Member or to join an existing AVC, an Alignment Conserver must post a recognition submission message publicly on the Maker Governance Forum.
 
 ##### 5.2.1.1
 
-A recognition submission message must be cryptographically signed by an Ethereum address containing at least 1 MKR, or that has delegated at least 1 MKR.
+A recognition submission message must be cryptographically signed by an Ethereum address containing at least 1 MKR, or that has delegated at least 1 MKR. This condition must be met during all mandate statuses, including 'active' and 'pending'.
```
Comment (0xldr) [inline MIP113/MIP113.md:319]: This change is not relevant at the point someone applies to become a member of an AVC. These statuses apply to the AVC, not the members.

File: MIP113/MIP113.md [hunk 3]
```diff
@@ -370,20 +370,20 @@ List of Unaffiliated AVC Members:
...
+An Unaffiliated AVC Member may join an AVC through the process described in MIP113 5.2.1.1 or start a new AVC through the process described in MIP101 2.5.3.
 
 #### 5.2.4: AVC Management
 
 The Governance Facilitators must monitor and record the status of each AVC.
 
 ##### 5.2.4.1
 
-An AVC is marked as _active_ status if it has fulfilled every requirement under *4.3* in the previous quarter.
+An AVC is marked as _active_ status if it has fulfilled every requirement under *MIP101 2.5.4* in the previous quarter.
```
Comment (0xldr) [inline MIP113/MIP113.md]:
```suggestion
An AVC is marked as _active_ status if it has fulfilled every requirement under *2.5.4* of The Atlas in the previous quarter.
```

File: MIP113/MIP113.md [hunk 4]
```diff
@@ -370,20 +370,20 @@ List of Unaffiliated AVC Members:
...
 
 ---
 ##### 5.2.4.2
 
-An AVC is marked as _inactive_ status if it has failed to fulfill the requirements under *4.3* for the most recent quarter, but has fulfilled the requirements for one or more prior quarters.
+An AVC is marked as _inactive_ status if it has failed to fulfill the requirements under *ATL101 2.5.4* for the most recent quarter, but has fulfilled the requirements for one or more prior quarters.
```
Comment (0xldr) [inline MIP113/MIP113.md]:
```suggestion
An AVC is marked as _inactive_ status if it has failed to fulfil the requirements under *2.5.4* of The Atlas for the most recent quarter, but has fulfilled the requirements for one or more prior quarters.
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

File: MIP113/MIP113.md (hunk 2 above)
Comment (0xNix0) [reply MIP113/MIP113.md:319]: Thanks for the reply. I wasn't clear enough, what I tried to specify is that AVC members should met the condition despite the status of their AVC being "active" or "pending".

**Comment by 0xldr** [comment]
Now outdated, multiple edits have been made.

**Pull request closed without merging**